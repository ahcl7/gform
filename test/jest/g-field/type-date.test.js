import { generateComponent } from "./common";

describe('GField with type date', function () {
  it('should render correctly', () => {
    const component = generateComponent({ key: 'date', type: 'input@datetime-local', flex: 'md-6' });
    expect(component.html()).toMatchSnapshot();
  });
});
