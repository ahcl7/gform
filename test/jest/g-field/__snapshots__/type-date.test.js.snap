// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GField with type date should render correctly 1`] = `
<div class="col-flex md-6 px-2">
  <div class="g-tf-wrapper g-tf  r">
    <pass-through class="input">
      <input autocomplete="off"
             class="g-tf-input"
             type="datetime-local"
             label="Date"
      >
    </pass-through>
  </div>
</div>
`;
